cdxev/__main__.py

```diff
@@ -709,6 +711,40 @@ def create_build_public_bom_parser(
     return parser
 
 
+# noinspection PyUnresolvedReferences,PyProtectedMember
+def create_init_sbom_parser(
+    subparsers: argparse._SubParsersAction,
+) -> argparse.ArgumentParser:
+    parser = subparsers.add_parser(
+        "init-sbom",
+        help=("Provides the first draft of a SBOM for manual completion"),
+    )
+    parser.add_argument(
+        "--name",
+        metavar="<schema path>",
+        help=("Name of the software the SBOM belongs to"),
+        default=None,
+        type=str,
+    )
+    parser.add_argument(
+        "--version",
+        metavar="<schema path>",
+        help=("Version of the software the SBOM belongs to"),
+        default=None,
+        type=str,
+    )
+    parser.add_argument(
+        "--supplier",
+        metavar="<schema path>",
+        help=("Name of the supplier of the software the SBOM belongs to"),
+        default=None,
+        type=str,
```

Comment:
I think it would be useful to have them in a group, as done for set with the ["targets"](https://github.com/Festo-se/cyclonedx-editor-validator/blob/51a64caf3fc66efe75798c22ed0acd159941ffb0/cdxev/__main__.py#L629).

Reply:
done